src/static/js/modules/explore-rates.js

```diff
@@ -563,362 +562,370 @@ function checkForJumbo() {
-    $loan.addOption({
-      'label': loanTypes[loan.type],
-      'value': loan.type,
-      'select': true
-    });
-    // if loan-type has changed as a result of the jumbo() operation, make sure everything is updated
+    params.isJumbo = true;
+    dropdown( 'loan-type' ).removeOption( jumbos );
+    dropdown( 'loan-type' ).enable( norms );
+    $loan.addOption( {
+      label:  loanTypes[loan.type],
+      value:  loan.type,
+      select: true
+    } );
+    // If loan-type has changed as a result of the jumbo() operation,
+    // make sure everything is updated.
     if ( loan.type !== params['loan-type'] ) {
-      params['prevLoanType'] = params['loan-type'];
+      params.prevLoanType = params['loan-type'];
       params['loan-type'] = loan.type;
-      dropdown('loan-type').disable( params.prevLoanType ).showHighlight();
-    }
-    else {
-      dropdown('loan-type').hideHighlight();
+      dropdown( 'loan-type' ).disable( params.prevLoanType ).showHighlight();
+    } else {
+      dropdown( 'loan-type' ).hideHighlight();
     }
-    // When the loan-type is agency or jumbo, disable conventional
-    if ( $.inArray( params['loan-type'], ['agency', 'jumbo'] ) >= 0 ) {
-      dropdown('loan-type').disable( 'conf' );
+    // When the loan-type is agency or jumbo, disable conventional.
+    if ( $.inArray( params['loan-type'], [ 'agency', 'jumbo' ] ) >= 0 ) {
+      dropdown( 'loan-type' ).disable( 'conf' );
     }
-    // Add links to loan messages
-    loan.msg = loan.msg.replace('jumbo (non-conforming)', '<a href="/owning-a-home/loan-options/conventional-loans/" target="_blank">jumbo (non-conforming)</a>');
-    loan.msg = loan.msg.replace('conforming jumbo', '<a href="/owning-a-home/loan-options/conventional-loans/" target="_blank">conforming jumbo</a>');
-    $('#hb-warning').removeClass('hidden').find('p').html( loan.msg );
+    // Add links to loan messages.
+    loan.msg = loan.msg.replace( 'jumbo (non-conforming)', '<a href="/owning-a-home/loan-options/conventional-loans/" target="_blank">jumbo (non-conforming)</a>' );
+    loan.msg = loan.msg.replace( 'conforming jumbo', '<a href="/owning-a-home/loan-options/conventional-loans/" target="_blank">conforming jumbo</a>' );
+    $( '#hb-warning' ).removeClass( 'hidden' ).find( 'p' ).html( loan.msg );
 
   } else {
-    params['isJumbo'] = false;
-    dropdown('loan-type').removeOption( jumbos );
-    dropdown('loan-type').enable( norms );
+    params.isJumbo = false;
+    dropdown( 'loan-type' ).removeOption( jumbos );
+    dropdown( 'loan-type' ).enable( norms );
 
-    $('#hb-warning').addClass('hidden');
+    $( '#hb-warning' ).addClass( 'hidden' );
     // Select appropriate loan type if loan was kicked out of jumbo
     if ( params.prevLoanType === 'fha-hb' ) {
-      $('#loan-type').val( 'fha' );
-
-    }
-    else if ( params.prevLoanType === 'va-hb' ) {
+      $( '#loan-type' ).val( 'fha' );
+    } else if ( prevLoanType === 'va-hb' ) {
+      $( '#loan-type' ).val( 'va' );
```

Comment:
I think these lines are unnecessary -- the next condition runs the same check but accesses `prevLoanType` on the params object.